centos_ci.sh

```diff
@@ -79,6 +79,11 @@ function prepare_ci_user() {
 }
 
 function setup_golang() {
+  # Install the required version of golang
+  curl -L -O https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz
+  tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz
+  export PATH=$PATH:/usr/local/go/bin
+
```

Comment:
The commit log does not explain why we cannot use the epel version. Might be worthwhile to file an epel bug as well?

Reply:
@cfergeau I am going to use same go version as we are using for other CI which is 1.12.x and which is part of internal build system.